zero/ner/main.py

```diff
@@ -116,7 +118,7 @@ def run(common_args, **task_args):
     torch.cuda.empty_cache()
 
     if args.do_eval:
-        dozen_path, luke_path, rgcn_path = get_saved_paths(args, tag="latest")
+        dozen_path, luke_path, rgcn_path = get_saved_paths(args, tag="last")
```

Comment:
This is a bug it seems like we've had for some time. There was an inconsistency issue between this line and L111 resulting in an error after training finished. Fixed here.